gui/wxpython/datacatalog/infomanager.py

```diff
@@ -0,0 +1,31 @@
+
+import wx
+import webbrowser
+
+from grass.script import gisenv
+
+
+class InfoManagerDataCatalog:
+    """
+    InfoBar Manager for Data Catalog
+    """
+    def __init__(self, infobar, sizer):
+        self.infoBar = infobar
+        self.sizer = sizer
+
+    def ShowInfoBar1(self, buttons):
+        self.sizer.Add(self.infoBar, wx.SizerFlags().Expand())
```

Comment:
Based on our discussion, the info bar should be at the top, but under the toolbar, so probably something like `...Insert(1, ...)` instead of plain `Add`.